backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -240,7 +246,7 @@ def resolve_fetch_failures_0_members(
         if success_cset_ids:
             try:
                 with get_db_connection(schema=schema, local=use_local_db) as con:
-                    concept_set_members__from_csets_and_members_to_db(con, success_cases)
+                    csets_and_members_to_db(con, success_cases, schema)
```

Comment:
## Bug fix: Finalized draft? --> Import expression item changes
    - Bug fix: After draft finalization, if expression items had changed since first fetch, would not update those expression items.

Previously, I was just updating `concept_set_members` for finalized drafts. In the following PR, I aim to update metadata as well (at the very least to remove [DRAFT] from the title):
- #836 

In this PR, I changed it so that new expression items will also be added! :)

This made me realize another bug that I still need to address, though:
- #839